### PR TITLE
Updated Stringtable.xml for German translation

### PR DESCRIPTION
**Pull request by xXGamerLionXx**
I have Updated Stringtable.xml for German people

## What type of PR is this.
1. [ ] Bug
2. [ ] Enhancement
3. [X] Translation

### What have you changed and why?
Information: Updated Stringtable.xml for German translation
    

### Please verify the following and ensure all checks are completed.

1. [X] Have you loaded the Mission in Singleplayer?
2. [X] Have you loaded the Mission in a Dedicated Server?

### Is further testing or are further changes required?
1. [X] No
2. [ ] Yes (Please provide further detail below.)

********************************************************
Notes:

### CODE REVIEW

**Comment by wurzel0701** [comment]
Edited the description to make proper use of the github markdown

File: A3-Antistasi/Stringtable.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
         <Polish />
         <Portuguese />
         <Russian>Загрузить предыдущее сохранение?</Russian>
-        <German />
+        <German> Vorherige Sitzung laden? </German>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Can you remove the Leading / Trailing whitespace please.

File: A3-Antistasi/Stringtable.xml [hunk 2]
```diff
@@ -21,96 +21,112 @@
         <English>HQ Options: Commander Only</English>
         <French>Options du QG : Commandants Uniquement</French>
         <Russian>Опции Штаба: Только для Командира</Russian>
+        <German> HQ-Optionen: Nur Commander</German>
```
Comment (Toakan) [inline A3-Antistasi/Stringtable.xml]: Whitespace here.

**Comment by Bob-Murphy** [comment]
@xXGamerLionXx Sorry that we did not merge it yet. This PR is not forgotten but delayed due to some other issues.

**Comment by xXGamerLionXx** [comment]
Yes is not a problem, I should have adapted the translation to the current one anyway. that's how it is to my knowledge.

File: A3-Antistasi/Stringtable.xml (hunk 1 above)
Comment (Toakan) [reply A3-Antistasi/Stringtable.xml]:
```suggestion
        <German>Vorherige Sitzung laden?</German>
```

File: A3-Antistasi/Stringtable.xml (hunk 2 above)
Comment (Toakan) [reply A3-Antistasi/Stringtable.xml]:
```suggestion
        <German>HQ-Optionen: Nur Commander</German>
```

**Comment by Bob-Murphy** [comment]
I need to manually merge them into the current stringtable as some originals changed since then.

**Comment by Bob-Murphy** [comment]
Updated by #940

**Pull request closed without merging**